File: src/main/java/no/nav/syfo/controller/domain/sykepengesoknad/Datospenn.java
package no.nav.syfo.controller.domain.sykepengesoknad;

import java.io.Serializable;
import java.time.LocalDate;

public class Datospenn implements Serializable {
    public LocalDate fom;
    public LocalDate tom;

    public Datospenn withFom(final LocalDate fom) {
        this.fom = fom;
        return this;
    }

    public Datospenn withTom(final LocalDate tom) {
        this.tom = tom;
        return this;
    }
}

File: src/main/kotlin/no/nav/syfo/controller/domain/sykmelding/Datospenn.kt
package no.nav.syfo.controller.domain.sykmelding

import java.io.Serializable
import java.time.LocalDate

class Datospenn : Serializable {
    var fom: LocalDate? = null
    var tom: LocalDate? = null
    fun withFom(fom: LocalDate?): Datospenn {
        this.fom = fom
        return this
    }

    fun withTom(tom: LocalDate?): Datospenn {
        this.tom = tom
        return this
    }
}
